### ISSUE TITLE
Idea_for_enhancements

### ISSUE THREAD

**Issue by addy369** [shreyko/SPACE-ENCOUNTERS#2]
When you are checking distance<50 criteria also ensure that the bullet s destroyed!!
Currently one bullet can kill multiple enemies(not an issue necessarily)

**Comment by shreyko** [comment]
yes so in the code the bullet does get reset after hitting the enemy. But I think that because the bullet velocity becomes very high, the player does not even notice and ends up firing twice. I will definitely turn bullet_vel down a notch and re-upload the code. 
Thanks for the comments.
